cmd/nerdctl/restart_linux_test.go

```diff
@@ -69,3 +72,33 @@ func TestRestartPIDContainer(t *testing.T) {
 
 	assert.Equal(t, baseOutput, sharedOutput)
 }
+
+func TestRestartWithTime(t *testing.T) {
+	t.Parallel()
+	base := testutil.NewBase(t)
+	tID := testutil.Identifier(t)
+
+	// here we do not use nginx-alpine because it restarts immediately after
+	// receiving the SIGTERM signal with ignoring the time specified
+	base.Cmd("run", "-d", "--name", tID, testutil.AlpineImage, "sleep", "infinity").AssertOK()
+	defer base.Cmd("rm", "-f", tID).AssertOK()
+	base.EnsureContainerStarted(tID)
+
+	inspect := base.InspectContainer(tID)
+	pid := inspect.State.Pid
+
+	// random time in the interval of [1, 20] to restart
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Can you keep the time as constant rather than a random time please?

Reply:
Sure.